etc/behat/services/pages.xml

```diff
@@ -53,5 +58,13 @@
         <service id="sylius.behat.page.checkout_thank_you" class="%sylius.behat.page.checkout_thank_you.class%" parent="sylius.behat.symfony_page" scope="scenario" public="false" />
 
         <service id="sylius.behat.page.external.paypal_checkout_express" class="%sylius.behat.page.external.paypal_checkout_express.class%" parent="sylius.behat.page" scope="scenario" public="false" />
+
+        <service id="sylius.behat.page.admin.country.index" class="%sylius.behat.page.admin.crud.index.class%" parent="sylius.behat.symfony_page" scope="scenario" public="false">
+            <argument type="service" id="sylius.behat.table_manipulator" />
```

Comment:
And then use with `parent` attribute so we don't have to inject this to every index page